package.json

```diff
@@ -1,6 +1,7 @@
 {
 	"private": true,
 	"name": "reactivesearch-packages",
+	"version": "1.0.0",
```

Comment:
@MrCode1997 We don't need this change.

Reply:
@bietkul Ok, perfect probably was a refuse. i can remove it.